packages/angular/src/date-time-picker/date-time-absolute.component.ts

```diff
@@ -29,25 +30,50 @@ import { format, getHours, getMinutes, setHours, setMinutes } from 'date-fns';
         <!-- tmp until we can implement a better time selector -->
         <div class="bx--form-item" style="margin-right: 1rem">
           <label class="bx--label">Start time</label>
-          <input ibmText type="time" [(ngModel)]="startTime" (change)="onChange()" />
+          <input ibmText type="time" [(ngModel)]="startTime" (change)="onChange()" theme="light"/>
         </div>
         <!-- tmp until we can implement a better time selector -->
         <div class="bx--form-item">
           <label class="bx--label">End time</label>
-          <input ibmText type="time" [(ngModel)]="endTime" (change)="onChange()" />
+          <input ibmText type="time" [(ngModel)]="endTime" (change)="onChange()" theme="light"/>
         </div>
       </div>
     </fieldset>
   `,
   styles: [
     `
+      /*
+        all of this is a bunch of gross styling hacks until we can settle on a reasonable
+        UX decision for the range picker. By default react forces the picker open, which
+        totally breaks the interaction for re-selecting dates. We also need to fix the HTML
+        structure upstream as we can't apply the right spacing (easily) due to the duplication
+        of classes at multiple levels of the underlying datepicker.
+      */
```

Comment:
is this related to more of an internal discussion or do we want to keep?

Reply:
Going to leave this for context, since it is some pretty gross styles I've had to add